tests/automation/user_actions.spec.ts

```diff
@@ -73,6 +73,9 @@ sessionTestTwoWindows(
       'module-conversation__user__profile-name',
       userB.userName,
     );
+    // he is a contact, close the new conversation button tab as there is no right click allowed on it
```

Comment:
Not sure what you mean here, no right click allowed for what action? Block? He is already an approved contact

Reply:
so this is the code
![image](https://github.com/oxen-io/session-playwright/assets/1544279/780888f3-ca44-44bc-90e0-191c8d102b17)

What happened before was that
you `create a contact`,
then click on `new-conversation-button`
this opens the tab with "choose private chat, create group or join community), and at the bottom of it, you can see your  contact lists.
then here 

```ts
 await waitForTestIdWithText(
      windowA,
      'module-conversation__user__profile-name',
      userB.userName,
    );
    ```
    you check that the user is indeed a contact
    
    and then you tried to right click on that contact, but before that contact item is not the conversation itself, you cannot block him from here. To do right click menu actions, you need the conversationListItem which is rendered on the "main list of conversations" which is another click on "new-conversation-button" (acts as back) away.

Reply:
[Screencast from 16-04-24 11:20:46.webm](https://github.com/oxen-io/session-playwright/assets/1544279/8ce8b2e9-a637-452a-a54d-6dd403a7dbb5)